lib/phoenix_live_view/js.ex

```diff
@@ -465,20 +465,20 @@ defmodule Phoenix.LiveView.JS do
   end
 
   @doc """
-  Toggles classes on an element.
+  Adds or removes element classes based on presence.
 
-    * `names` - The string of classes to add.
+    * `names` - A string with one or more class names to toggle.
```

Comment:
Changing the `the`s here and below makes this somewhat inconsistent with the other functions in this module, e.g.  `toggle`, `add_class`, etc.

Reply:
That's true. I wanted to keep the changes contained but noticed the inconsistency. I'll go ahead and update the rest.

Reply:
Done